index.js

```diff
@@ -781,10 +781,10 @@ Command.prototype.optionMissingArgument = function(option, flag) {
  * @api private
  */
 
-Command.prototype.unknownOption = function(flag) {
+Command.prototype.unknownOption = function(unknown) {
```

Comment:
So, if we change the function name, it'll be considered as breaking change, and should we deferred until the next major release.